3-extra/1-factorial.js

```diff
@@ -8,9 +8,9 @@
     Using a loop, complete the function below so it returns the factorial of the number being passed in. 
 */
 
-function factorial(input) {
-    // TODO
-}
+
+    const factorial = (input) => (n > 0 ? n * factorial(--n) : 1);
+
```

Comment:
Great work Iwona. I hope you don't mind me leaving comments on your pull request. I'm catching up on code review homework. I like how you solved this, especially using arrow functions. Nice